packages/framework/pr_tools/PullRequestLinuxDriver.sh

```diff
@@ -8,6 +8,7 @@ source ${SCRIPTPATH:?}/common.bash
 on_weaver=$(echo "$@" | grep '\-\-on_weaver' &> /dev/null && echo "1")
 on_ats2=$(echo "$@" | grep '\-\-on_ats2' &> /dev/null && echo "1")
 bootstrap=$(echo "$@" | grep '\-\-\no\-bootstrap' &> /dev/null && echo "0" || echo "1")
+kokkos_develop=$(echo "$@" | grep '\-\-kokkos\-develop' &> /dev/null && echo "0" || echo "1")
```

Comment:
I think this is backwards.  Note that the bootstrap variable is inverted from its option (--no-bootstrap)

Reply:
Good catch! 😅 

I pushed the change: re-organized the lines to group them by the way the flag is parsed and renamed to `on_kokkos_develop` to copy the convention of the other variables.